include/openPMD/IO/IOTask.hpp

```diff
@@ -238,15 +238,15 @@ class IOTask
            Parameter< op > const& p)
             : writable{w},
               operation{op},
-              parameter{new Parameter< op >(p)}
```

Comment:
that abstract base class should not be copied to each other. That is object slicing:
https://stackoverflow.com/questions/274626/what-is-object-slicing

Reply:
Hm, but since we explicitly new on the `Parameter< op >` this should not be slicing...
At least `clang-tidy-6.0` does not find it as such:
```bash
clang-tidy-6.0 test.cpp -checks='*,cppcoreguidelines-slicing' --
```

Reply:
Nah, it is likely still slicing since we `push` `IOTasks` into `std::qeues` which are copies of `AbstractParameter`